create_jenkins_job.py

```diff
@@ -81,6 +81,7 @@ def main(argv=None):
         'use_isolated_default': 'true',
         'build_args_default': '--event-handlers console_cohesion+ console_package_list+ --cmake-args -DINSTALL_EXAMPLES=OFF -DSECURITY=ON',
         'test_args_default': '--event-handlers console_direct+ --executor sequential --retest-until-pass 10',
+        'compile_on_clang_default': 'false',
```

Comment:
"compile on" doesn't sound write to me. Code is compiled with or by a compiler. So I would suggest renaming this everywhere to "compile with".

Reply:
Makes sense, changed the name